channels/pe.m3u

```diff
@@ -127,3 +127,11 @@ https://5ee0faac3bbae.streamlock.net/visionnoticias/visionnoticias/playlist.m3u8
 https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
 #EXTINF:-1 tvg-id="Willax.pe" tvg-name="Willax" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="General",Willax
 https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/WillaxTV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/rwtelevision/picture?width=320&height=320" group-title="Local",RW Televisión (Tarapoto) [Not 24/7]
+https://tvdatta.com:3952/live/rwtelevisionlive.m3u8
```

Comment:
Broken Link

Reply:
I agree, same on my side.

Reply:
The link works

Reply:
The Via Televisió channel had the same problem, so I'm going to put [Not 24/7]. The peruvian issue aobut streaming is very limited compared to other countries.